README.md

```diff
@@ -1 +1,92 @@
-# pdfgen
\ No newline at end of file
+# pdfgen
+
+An API for creating PDF files both from static HTML and Thymeleaf templates.
+
+### System requirements ###
+
+
+This API requires `wkhtmltopdf` installed on your machine. It should be added to system path. You can download it from [here](https://wkhtmltopdf.org/).
+
+
+Basic usage
+---------------
+
+The following call will produce basic PDF file converted from HTML:
```

Comment:
The following call will convert the provided HTML document and produce a PDF file